qa/tasks/vstart_runner.py

```diff
@@ -777,57 +778,21 @@ def __init__(self):
         # methods to work though.
         self.pools = {}
 
+        self.testdir = None
```

Comment:
ditto

Reply:
Marking resolved due to https://github.com/ceph/ceph/pull/38481#discussion_r592928680.